nixos/modules/virtualisation/qcow.nix

```diff
@@ -0,0 +1,66 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.virtualisation.qcow;
+in
+{
+  options = {
+    virtualisation.qcow = {
+      baseImageSize = mkOption {
+        type = with types; either (enum [ "auto" ]) ints.positive;
+        default = "auto";
+        example = 8192;
+        description = ''
+          The base size of the QCOW2 image in MiB.
+        '';
+      };
+      additionalImageSpace = mkOption {
+        type = types.int;
+        default = 0;
```

Comment:
I got the idea to put it on 0 by default because of this: https://github.com/NixOS/nixpkgs/blob/c423a379bbe2e6c479087b6008962d8a35b89e1c/nixos/modules/virtualisation/qemu-vm.nix#L280

Reply:
huh crazy, that could break sometimes :)

Reply:
So what do you say, 0 or 512?

Reply:
I would say 512, since its the default from make-disk-image. and would be weird to have a new default in a new layer

Reply:
Done,